packager/packager.livecodescript

```diff
@@ -2982,3 +2993,133 @@ private function _printCharXTimes pChar, pTimes
   end repeat
   return tStr
 end _printCharXTimes
+
+/*
+
+To run the packager on the command line of an installed IDE use:
+
+```
+<path to IDE exe> \
+  -ui \
+  <path to levure/packager/packager.livecodescript> \
+  <path to app/standalone.livecode> \
+  <buildprofile>
+```
+
+For example:
+
+```
+"/Applications/LiveCode 9.6.8.app/Contents/MacOS/LiveCode" \
+  -ui \
+  ~/myapp/levure/packager/packager.livecodescript \
+  ~/myapp/app/standalone.livecode \
+  beta
+```
+*/
+on startup
+   _LoadHomeStack
+
+  start using stack "revSBLibrary"
+
+  if there is no stack $1 then
+    write "Stack not found" && $1 & return to stderr
+    quit 1
+  end if
+
+  go stack $1
+
+  put "running" into sCommandLineStatus
+
+  send "_doPackageCommand $1, $2" \
+    to me \
+    in 100 milliseconds
+
+  /* There is a send in time in levurePackageApplication so we need to wait
+   * here until complete */
+  wait while sCommandLineStatus is "running" with messages
+
+  switch sCommandLineStatus
+  case "complete"
+    /* completed ok but there may be warnings */
+    if revStandaloneGetWarnings() is not empty then
```

Comment:
@trevordevore currently in this case the command will have actually failed at the moment because the standalone builder doesn't suppress the build warning dialog correctly when `revSetTestEnvironment` is set to true.  However, I have created an issue to resolve that. Not an issue if the standalone builder can find all your resources etc though.